Payments/Operation/02_ManagePaymentsAndCaptures.md

```diff
@@ -0,0 +1,103 @@
+
+You can use this procedure to display all incoming references to a payment and capture transaction from a payment service provider. You can check for communication errors between Actindo and the payment service provider that has been caused in a termination of a workflow, for example.   
+If you cannot deliver the product and want to give the money back to the customer, you can manually change the status of a transaction to "Refund" to initiate a repayment. Additionally, you can set the status to "Void" to clear the database from faulty entries.      
+
+If everything works fine with the posting of a capture, you do not need to monitor the *Payments and Captures* menu entry.   
+> [Info] You can only change the status of a payment and capture transaction, but not any data.
+
+
+## Check payment and capture transaction
+
+A capture provides numerous payment-related details, which may vary depending on the payment service provider settings. To decide, whether you can manually refund or void a payment and capture transaction, you need to check its details.
+
+#### Prerequisites
+
+- At least one connection has been created, see [Create a connection](../Integration/01_ManageConnections.md#create-a-connection).
+- At least one payment and capture transaction has been created.
+- It is recommended that you add the *Status ID* column to your view so that you are able to sort or filter for specific statuses. 
+
+#### Procedure
+
+*Payments > Payments and Captures > Tab LIST*  
+
+![Payment and capture transactions](../../Assets/Screenshots/Payments/PaymentsCaptures/LISTPaymentsCaptures.png "[Payment and capture transactions]")  
+
+
+1. Check the payment and capture transaction status in the *Status* column. The status displays the current stage in the payment process. You can use the status ID prefixed below to filter the list. The following statuses are available:   
+    - **1 - Preparing**  
+        The transaction has been created, but is not yet transferred.
+    - **2 - Unconfirmed**   
+        A refund is reserved but not yet released by an Actindo user. This status is relevant for some functions only, for example, the cancellations and chargebacks.
+    - **3 - Success**  
+        Actindo has got the response from the service payment provider, that the transfer was successfull.
+    - **4 - Failure**   
+        The payment service provider has received the request, but it cannot process it. If available for the connection, the *Status information* field provides further information.
+    - **5 - Error**   
+       Errors have been occurred during transfer.
+    - **6 - Void**   
+       The transaction has been voided, see [Void payment and capture transaction](#void-payment-and-payment-and-capture-transaction).
+    
+2. Click on a transaction to see the details.   
```

Comment:
```suggestion
2. Click a capture and payment transaction to see the details.   
```

Reply:
Das verwirrt nur, ich bin ja nun in der payment und capture